examples/pytorch/ogb/directional_GSN/README.md

```diff
@@ -0,0 +1,55 @@
+# directional_GSN
+
+## Introduction
+
+This is an example of implementing [directional_GSN](https://arxiv.org/abs/2006.09252) for graph classification in DGL.
+
+directional_GSN is a combination of GSN with DGN, where we defined a vector field based on structural features instead of eigenvectors.
```

Comment:
`eigenvectors` of what?

Also strictly speaking, if you mean eigenvectors of the adjacency matrix and Laplacian matrix, they are also structural features themselves.